### DISCUSSION TITLE
Is it possible to set my GH issue tracker read-only?

### DISCUSSION THREAD

**Discussion by Dead-maksim** [community/community#64600] [Repositories] [answered]
### Select Topic Area

Question

### Body

I have a GitHub project with several(70+) issues. I'll move my project and all issues from GitHub to my site with a new issue page. What I want to do but don't know how:

Stay all issues are visible but closed to posting. Is this possible? I can only hide all issues by unchecking the "issue" checkbox in settings.

How can I notify users about moving the issue tracker from GitHub to the issue page? (perfect option will be a pop-up window, but it is impossible) I understand I can write ads and links to README.md, but users can go directly to the issue page ignoring readme. Also, I know that I can change the issue template, but I'm afraid users will be ignoring my templates (like always)

**Comment by arafajs** [comment]
- Go to the repository settings page.
- Under the "Features" section, uncheck the "Issues" checkbox.
- Click the "Save" button.

**Comment by Dead-maksim** [comment]
Hi @arafajs . Thank you for your advice. I prefer to keep my issues available for read-only. Is it possible?

**Reply by arafajs** [reply, marked as answer]
Sorry. GitHub does not have a built-in way to make issues read-only.

**Comment by zbynekwinkler** [comment]
To make issues read-only add `.github/ISSUE_TEMPLATE/config.yml` and make sure to set `blank_issues_enabled` to `false` and don't add any templates. For more info see [Configuring the template chooser](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#configuring-the-template-chooser) or [Using External Links as GitHub Issue Template Options](https://ryanmulligan.dev/blog/external-links-issue-template-options/).

**Reply by john-science** [reply]
Does this hack still work? I kind of love it.

**Reply by zbynekwinkler** [reply]
Yes, it does.
